WordPress/Jetpack/Classes/Utility/DataMigrator.swift

```diff
@@ -13,18 +16,32 @@ protocol ContentDataMigrating {
     func deleteExportedData()
 }
 
-enum DataMigrationError: LocalizedError {
-    case databaseCopyError
+enum DataMigrationError: LocalizedError, CustomNSError {
+    case databaseImportError
+    case databaseExportError
     case sharedUserDefaultsNil
     case dataNotReadyToImport
 
     var errorDescription: String? {
         switch self {
-        case .databaseCopyError: return "The database couldn't be copied to/from shared directory"
+        case .databaseImportError: return "The database couldn't be copied from shared directory"
+        case .databaseExportError: return "The database couldn't be copied to shared directory"
         case .sharedUserDefaultsNil: return "Shared user defaults not found"
         case .dataNotReadyToImport: return "The data wasn't ready to import"
         }
     }
+
+    static var errorDomain: String {
+        return String(describing: DataMigrationError.self)
+    }
+
+    var errorUserInfo: [String: Any] {
+        var userInfo = [String: Any]()
+        if let errorDescription {
+            userInfo[NSDebugDescriptionErrorKey] = errorDescription
+        }
+        return userInfo
+    }
```

Comment:
Added to easily identify errors in Sentry. Read more [here](https://docs.sentry.io/platforms/apple/guides/ios/usage/).